lib/CustomView/Config.php

```diff
@@ -69,6 +69,25 @@ private static function getRepository()
         return self::$locationAwareConfigRepository;
     }
 
+    /**
+     * @param mixed $data
+     * @return array
+     */
+    protected static function flipArray(mixed $data): array {
+        if(empty($data['classes'])) {
+            return [];
+        }
+        else {
+            $flipArray = [];
+            $tempClasses = explode(',', $data['classes']);
+
+            foreach ($tempClasses as $tempClass) {
+                $flipArray[$tempClass] = null;
+            }
+            return $flipArray;
```

Comment:
I think [this one](https://www.php.net/manual/en/function.array-fill-keys.php) could work:

```suggestion
            $tempClasses = explode(',', $data['classes']);
            return array_fill_keys($tempClasses, null);
```

Reply:
It does, neat, thanks!